docs/getting-started/use-cases/computervision/annotation/cvat/cvat_quick_guide.md

```diff
@@ -67,7 +67,11 @@ You can also bring your own labeling tool as a reproducible template in Onepanel
 
 12. Click **Job #1** to go into CVAT to start annotating your data. See [CVAT's user guide](https://github.com/opencv/cvat/blob/develop/cvat/apps/documentation/user_guide.md#interface-of-the-annotation-tool) for more information on the annotation tool interface.
   :::note
-  CVAT requires super user permission to perform certain tasks. Onepanel automatically creates a superuser when you create a new CVAT workspace. Your username will be `admin` and password will be onepanel token. Note that you generated auth token for Onepanel while setting up the Onepanel. [See step #9](/docs/getting-started/quickstart#step-1-install-onepanel) of installation guide.
+  CVAT requires super user permission to perform certain tasks. Onepanel automatically creates a superuser 
```

Comment:
Don't think we need to call out this out anymore. The system automatically creates the users with the correct permissions. We can refer them to the CVAT guide around user management instead.